cli/gas.c

```diff
@@ -63,6 +72,17 @@ static void print_line(unsigned long addr, uint8_t *bytes, size_t n)
 	printf("|\n");
 }
 
+static void gas_dump_sig_handler(int signum)
+{
+	if (signum == SIGTERM) {
```

Comment:
No, this is not a good idea. If someone kills the process at just the wrong time they will get this odd error message.